src/ode.pyx

```diff
@@ -15,16 +15,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
+# distutils: define_macros=NPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION
+
 import numpy as N
 cimport numpy as N
-from timeit import default_timer as timer
-
 import itertools
 import multiprocessing
+from timeit import default_timer as timer
 
-from exception import ODE_Exception, AssimuloException
-from problem import Explicit_Problem, Delay_Explicit_Problem, Implicit_Problem, SingPerturbed_Problem
-from support import Statistics
+from assimulo.exception import ODE_Exception, AssimuloException
+from assimulo.problem import Explicit_Problem, Delay_Explicit_Problem, Implicit_Problem, SingPerturbed_Problem
+from assimulo.support import Statistics
```

Comment:
Do we know by the way what change in the Python version caused these imports to fail? I.e. why we needed to update them? Just so we understand it on our end.

Reply:
I believe this to be due to this: [https://cython.readthedocs.io/en/latest/src/userguide/migrating_to_cy30.html#namespace-packages](https://cython.readthedocs.io/en/latest/src/userguide/migrating_to_cy30.html#namespace-packages)

Reply:
Great, thank you for sharing!